Pod/Classes/RZMessagingWindow.h

```diff
@@ -0,0 +1,154 @@
+ */
+@interface RZMessagingWindow : UIWindow
+
+/**
+ *  Set this to tell the window what class of ViewController it should initialize.  A standard 
+ *  [[Class alloc] init] will be called on it and added to the subview and the viewController.
+ */
+@property (assign, nonatomic) Class messageViewControllerClass;
+
+
+/**
+ *  The Message that is currently being displayed.
+ */
+@property (nonatomic, readonly) NSError *displayedError;
+
+/**
+ *  Returns a flag if the window is currently displaying an error, presenting and error, or dismissing.
+ */
+@property (nonatomic, readonly) BOOL isCurrentlyDisplayingAnError;
+
+/**
+ *  Designated Initializer.  This does not return a shared instance.  Should only be called once
+ *  unless you want multiple instances of the window.
+ *
+ *  @return The RZMessagingWindow instance.
+ */
++ (instancetype)messagingWindow;
+
+/**
+ *  Returns a messagingWindow configured with default colors and to use RZErrorMessagingViewController
+ *
+ *  @return fully configured RZMessagingWindow
+ */
++ (instancetype)defaultMessagingWindow;
+
+/**
+ *  Shows a message from an NSError object.  This object will be passed to the messageVC that you
+ *  provide in the viewConfigurationBlock.
+ *
+ *  @param error The error containing your message information.
+ */
+- (void)showMessageFromError:(NSError *)error;
+
+/**
+ *  Shows a message from an NSError object.  The object will be passed to the messageVC that you 
+ *  provide in the ViewConfigurationBlock.
+ *  The display strength will determin what interaction needs to be taken to dismiss the Message
+ *  and how it will be animated.
+ *
+ *  @param error    The error container the message information.
+ *  @param strength The strength of the message.
+ *  @param animated if the presentation should be animated or not.
+ */
+- (void)showMessageFromError:(NSError *)error strength:(RZMessageStrength)strength animated:(BOOL)animated;
+
+/**
+ *  Hides a displayed message if it is currently being shown, or removes it from the queue if it
+ *  hasen't made it to the screen yet.  If nil is passed in for the error object, the first message
+ *  will automatically be removed.
+ *
+ *  @param error    The error pertaining to the message to be removed.
+ *  @param animated if the hide should be animated or not.
+ */
+- (void)hideMessage:(NSError *)error animated:(BOOL)animated;
+
+/**
+ *  Hides all messages and clears out the entire queue.
+ *
+ *  @param animated if the removal should be animated or not.
+ */
+- (void)hideAllMessagesAnimated:(BOOL)animated;
+
+/**
+ *  Returns the strength of the displayed error message.  If no error message will return weak.
+ *
+ *  @return The message strength of the displayed error
+ */
+- (RZMessageStrength)strengthOfDisplayedError;
+
+@end
+
+#pragma mark - RZRootMessagingViewController
+
+/**
+ *  Root messaging View Controller.  Very simple implementation.
+ */
```

Comment:
Perhaps mentions that this is the class of the `RZMessagingWindow`'s root view controller